src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,62 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int numberOfFigures = 4; // start count from zero
+    public static final int maxValuesOfUnits = 10;
+    public static final int minValuesOfUnits = 1;
```

Comment:
```suggestion
    private static final int NUMBER_OF_FIGURES = 4;
    private static final int UPPER_BOUND = 10;
```
you are not a developer if you count not from 0 :)
There is no need for min value